.travis.yml

```diff
@@ -4,16 +4,27 @@ language: rust
 os: osx
 osx_image:
   - xcode10.1
-before_install:
-  - brew install sqlcipher --with-fts
 rust:
   - stable
   - beta
   - nightly
+cache: cargo
 matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
+env:
+  global:
+    - FRAMEWORK_NAME=ApplicationServices.framework.zip
```

Comment:
looks good, i wonder if this will grow in size at some point where it will be too big, but maybe that's for another day.